src/Spotify.js

```diff
@@ -0,0 +1,15 @@
+import { URL } from 'url';
+
+export const shouldTransform = url => {
+  const { host, pathname } = new URL(url);
+
+  return (
+    (host === 'open.spotify.com' || host === 'www.open.spotify.com') &&
```

Comment:
well, i guess "www.open.spotify.com" is not valid url.

Reply:
Hey, thanks for pointing I forgot to remove this earlier I had written a test as well but removed it as it's invalid. Fixing it right away.